src/components/Header/index.js

```diff
@@ -97,21 +105,29 @@ const socialArr = [
 const SocialLinks = () => (
   <div className={styles.social}>
     {socialArr.map(({ name, component, href }) => (
-      <a key={name} className={styles.socialLink} href={href} target="_blank">
+      <a
+        key={name}
+        className={styles.socialLink}
+        href={href}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
         {component}
       </a>
     ))}
   </div>
 );
 
-const BottomNav = () => (
+const BottomNav = ({ qtyProduct, history }) => (
   <div>
-    <div className={styles.btnItem}>
+    <div className={styles.btnItem} onClick={() => history.push("/cart")}>
```

Comment:
use Link component for this (https://reacttraining.com/react-router/web/api/Link)